### PR TITLE
add create partition obj

### PR DESCRIPTION
**Pull request by tuanad121**
add a function to create partition obj from list of (start time, end time, label)

### CODE REVIEW

**Review by j9ac9k — CHANGES_REQUESTED** [review]
There is a lot of duplicate code in `create_partition_obj` and 'read_lab'; please refactor such that those two methods can use the same code for the same task.

Also, I would recommend renaming `create_partition_obj` to `create`.

**Comment by tuanad121** [comment]
will do it soon :D

File: signalworks/tracking/partition.py [hunk 1]
```diff
@@ -273,48 +274,10 @@ def read_lab(
...
-        # value = numpy.array(value, dtype='U16' if label_type is str else numpy.float64)
-        value = numpy.array(
-            value, dtype="U16"
-        )  # if label_type is str else numpy.float64)
-        return Partition(
-            time, value, fs=fs, path=name
-        )  # u1p to 16 characters (labels could be words)
+            t1 = float(tmp1)  # in second
+            t2 = float(tmp2)  # in second
+            data.append((t1 * 1000, t2 * 1000, label))
```
Comment (j9ac9k) [inline signalworks/tracking/partition.py]: Why are we multiplying by 1,000 here?  Don't we want to keep it in "seconds"?

**Comment by j9ac9k** [comment]
wait, why are you bumping version on this branch?

...and why is the bump the minor version, and not the patch version?

**Comment by tuanad121** [comment]
> wait, why are you bumping version on this branch?
> 
> ...and why is the bump the minor version, and not the patch version?

I built the release from the branch so we can use new functions of `Partition` in Fred. The version on Pypi is 0.2.0 but I'm not sure why the version of branch is still < 0.2.0. So I did minor bump instead of several patch bump to update the version.

File: signalworks/tracking/partition.py (hunk 1 above)
Comment (tuanad121) [reply signalworks/tracking/partition.py]: In the lab file, the timestamp is in second. In our forced file, it's in millisecond. So I use millisecond for `create` function. :D

Comment (j9ac9k) [reply signalworks/tracking/partition.py]: The `forced` file is part of a downstream issue that should not be the concern of signalworks, let's keep the units consistent with signalworks and stick with `samples` and  `seconds`; we can let Fred (or other downstream tools) handle unit conversions as they need to.

**Comment by j9ac9k** [comment]
This isn't made explicit in the README (I should make it so); but the version numbers (major, minor, patch) do have definitions for what a version change entails, you can read about semantic versioning here: https://semver.org/

The needs of a downstream tool (such as Fred), should not cause us to generate premature releases here with signalworks.  If we need to tap into a non-standard branch for a downstream tool, we can do so just as we currently do with pyqtgraph.

We should likely implement a change where only the CI deploys code to pypi, with a git version tag being committed to the master branch.

File: signalworks/tracking/partition.py (hunk 1 above)
Comment (tuanad121) [reply signalworks/tracking/partition.py]: okay, will do so

**Review by j9ac9k — APPROVED** [review]

**Pull request merged**